R/stat-summary.r

```diff
@@ -270,13 +270,13 @@ mean_sdl <- wrap_hmisc("smean.sdl")
 #' @rdname hmisc
 median_hilow <- wrap_hmisc("smedian.hilow")
 
-#' Calculate mean and standard error
+#' Calculate mean and standard error of the mean
 #'
 #' For use with [stat_summary()]
 #'
-#' @param x numeric vector
-#' @param mult number of multiples of standard error
-#' @return A data frame with columns `y`, `ymin`, and `ymax`.
+#' @param x numeric vector.
+#' @param mult number of multiples of standard error.
+#' @return A data frame with columns `y` with the `mean`, `ymin` with the mean minus the standard error, and `ymax`, the mean plus the standard error.
```

Comment:
Preserve text width of 80 characters

```suggestion
#' @return A data frame with columns `y` with the `mean`, `ymin` with the mean
#' minus the standard error, and `ymax`, the mean plus the standard error.
```